apps/aechannel/test/aesc_txs_SUITE.erl

```diff
@@ -150,7 +150,10 @@
          fp_missing_contract_address/1,
 
          fp_insufficent_tokens/1,
-         fp_too_soon/1
+         fp_too_soon/1,
+
+         % off-chain name registration not allowed
```

Comment:
I see the related spec is

https://github.com/aeternity/protocol/blame/d081b9bacd10a3eeaf680902f9258c76e6e162e7/channels/OFF-CHAIN.md#L1041-L1042